src/community/policies.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: section
+title: "Policies"
+permalink: /community/policies/
+section_menu: section-menu/community.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Policies
+
+This page contains list of major policies agreed by the Apache Beam community.
+
+## Releases
+
+Apache Beam makes minor releases every 6 weeks. Apache Beam has a [calendar](https://calendar.google.com/calendar/embed?src=0p73sl034k80oob7seouanigd0%40group.calendar.google.com&ctz=America%2FLos_Angeles) for cutting the next release branch. Once a release branch is cut community works on quickly finalizing that release.
```

Comment:
is cut community works on quickly finalizing --> is cut, the community works quickly to finalize